src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SNI/SNIMarsHandle.cs

```diff
@@ -239,16 +253,18 @@ private uint SendPendingPackets()
 
                                 if (result != TdsEnums.SNI_SUCCESS && result != TdsEnums.SNI_SUCCESS_IO_PENDING)
                                 {
-                                    SqlClientEventSource.Log.TrySNITraceEvent("<sc.SNI.SNIMarsHandle.SendPendingPackets |SNI|ERR> InternalSendAsync result is not SNI_SUCCESS and is not SNI_SUCCESS_IO_PENDING");
+                                    SqlClientEventSource.Log.TrySNITraceEvent("SNIMarsHandle.SendPendingPackets | SNI | ERR | MARS Session Id {0}, InternalSendAsync result is not SNI_SUCCESS and is not SNI_SUCCESS_IO_PENDING", ConnectionId);
                                     return result;
                                 }
 
                                 _sendPacketQueue.Dequeue();
+                                SqlClientEventSource.Log.TrySNITraceEvent("SNIMarsHandle.SendPendingPackets | SNI | INFO | MARS Session Id {0}, _sendPacketQueue dequeued, count ", ConnectionId, _sendPacketQueue?.Count);
```

Comment:
Missing parameter reference in the string?
```suggestion
                                SqlClientEventSource.Log.TrySNITraceEvent("SNIMarsHandle.SendPendingPackets | SNI | INFO | MARS Session Id {0}, _sendPacketQueue dequeued, count {1}", ConnectionId, _sendPacketQueue?.Count);
```